hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/OMRequestTestUtils.java

```diff
@@ -1571,7 +1571,7 @@ public static long addParentsToDirTable(String volumeName, String bucketName,
               OMRequestTestUtils.createOmDirectoryInfo(pathElement, ++objectId,
                       parentId);
       OMRequestTestUtils.addDirKeyToDirTable(true, omDirInfo,
-              volumeName, bucketName, txnID, omMetaMgr);
+              volumeName, bucketName, ++txnID, omMetaMgr);
```

Comment:
Is this change(`++txnID`) required?

Reply:
From txn ID point of view, ID should be different for every update operation. Although here in the test, previous same ID doesn't cause any problem since it's not checked in the test.  But if later, some new tests start to check the ID, then they will have problems.